src/api/apiKey.js

```diff
@@ -29,9 +29,9 @@ function deleteApiKeyProperty_ () {
 }
 
 function obfuscateApiKey_ (key) {
-  return key.replace(/(.{3})(.+)(.{3})/g, function (_match, start, middle, end) {
-    return start + Array(middle.length).join('*') + end
-  })
+  return key.replace(/(.{3})(.+)(.{3})/g, (_match, start, middle, end) =>
+    start + Array(middle.length).join('*') + end
```

Comment:
You should return here

Reply:
Aren't the arrow function expressions making the return obsolete?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions